$MirOS: ports/devel/omniORB/patches/patch-include_omnithread_h,v 1.1 2008/08/13 10:02:02 tg Exp $
--- include/omnithread.h.orig	Fri Sep 19 09:28:40 2008
+++ include/omnithread.h	Sun Nov  2 03:31:22 2008
@@ -157,6 +157,12 @@ class omni_thread;
 #elif defined(__freebsd__)
 #include <omnithread/posix.h>
 
+#elif defined(__midnightbsd__)
+#include <omnithread/posix.h>
+
+#elif defined(__mirbsd__)
+#include <omnithread/posix.h>
+
 #elif defined(__netbsd__)
 #include <omnithread/posix.h>
 
